R/algorithm.R

```diff
@@ -35,6 +35,18 @@ algorithm <- function() {
       logic = "atc =~ '^A10' AND NOT (atc =~ '^(A10BJ|A10BK01|A10BK03)')",
       comments = "GLP-RAs or dapagliflozin/empagliflozin drugs are not kept."
     ),
+    is_non_insulin_gld_code = list(
+      register = "lmdb",
+      title = "Non-insulin glucose-lowering drugs",
+      logic = "NOT (atc =~ '^A10B' OR atc =~ '^A10A' OR atc =~ '^A10AE5')",
```

Comment:
I added `atc = "A10A"` to this that was not part of the original. @Aastedet what do you think? Otherwise, it is possible that both is insulin and is non-insulin can be both true, is that possible?

Reply:
With the following, there should be no overlap of what we (for the purpose of diabetes type-classification) define as insulins:
```suggestion
      logic = "atc =~ '^A10B' OR atc =~ '^A10AE5')",
```
vs. non-insulins (see below)